doc/api/assert.md

```diff
@@ -301,6 +301,38 @@ assert.ifError(new Error());
 // Throws Error
 ```
 
+## assert.isInstanceOf(obj, constructor, message)
+<!-- YAML
+added: v7.8.1
```

Comment:
This would better use `added: REPLACEME`, that placeholder gets replaced with the actual version number by the releaser